client/js/condensed.js

```diff
@@ -33,5 +34,6 @@ function updateText(condensed, addedTypes) {
 			text += obj[messageType] > 1 ? "s" : "";
 		}
 	}
-	condensed.find(".condensed-msg").text(text);
+	condensed.find(".condensed-text")
+		.html(text + templates.msg_condensed_toggle());
```

Comment:
So bounding box would roughly be:

<img width="169" alt="28763185-2e2aba90-758a-11e7-8933-c7b8a40f9d44" src="https://user-images.githubusercontent.com/113730/29105177-1a413ebc-7c99-11e7-8391-77d526019fe8.png">

(I quickly edited this with Mac's preview, not set background on actual link, so *picture may differ from actual product* 😅)